bigbluebutton-html5/imports/ui/components/settings/submenus/notification/component.jsx

```diff
@@ -1,6 +1,7 @@
 import React from 'react';
 import Toggle from '/imports/ui/components/common/switch/component';
 import { defineMessages, injectIntl } from 'react-intl';
+import audioManager from '/imports/ui/services/audio-manager';
```

Comment:
```suggestion
import AudioManager from '/imports/ui/services/audio-manager';
```

It's a singleton - uppercase to keep the usage pattern seen elsewhere.